mods/dispatcher/src/dispatcher.ts

```diff
@@ -18,11 +18,16 @@ if (process.env.NODE_ENV === "dev") {
 
 function dispatch(channel: any) {
   try {
+    const maryTTSConfig = {
```

Comment:
In this case, you will have to use variables from the environment instead of hardcoded values.